int32 address
bool value
---
bool ret 
